examples/features/proto/echo/echo.proto

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+package echo;
```

Comment:
`grpc.examples.features.proto.echo` ?
`grpc.examples.echo` ?

Reply:
`grpc.examples.echo` should be good.

I didn't add `option go_package = `, I think we don't need it for this.

Reply:
And the directory name needs to be renamed to `grpc_examples_echo`, to match the go package name.